src/main/kotlin/org/opensearch/commons/notifications/NotificationConstants.kt

```diff
@@ -36,7 +36,6 @@ object NotificationConstants {
     const val NAME_TAG = "name"
     const val DESCRIPTION_TAG = "description"
     const val IS_ENABLED_TAG = "is_enabled"
-    const val FEATURE_LIST_TAG = "feature_list"
```

Comment:
What about removing `FEATURE_CONFIG_LIST_TAG`?

Reply:
Same as my comment to Drew.